typescript/db/src/repository/label.ts

```diff
@@ -2,30 +2,45 @@ import { PrismaClient, Prisma } from "@prisma/client";
 import { Repository, DbLabel } from "@labelflow/common-resolvers";
 
 import { castObjectNullsToUndefined } from "./utils";
+import { checkUserAccessToDataset } from "./access-control";
 
 const prisma = new PrismaClient();
 
-export const countLabels: Repository["label"]["count"] = async (where) => {
-  if (!where) {
+export const countLabels: Repository["label"]["count"] = async (
+  whereWithUser
+) => {
+  if (!whereWithUser) {
     return 0;
   }
+  const { user, ...where } = whereWithUser;
   if ("datasetId" in where) {
```

Comment:
same here, shouldn't we check that the user has access to the dataset?

Reply:
Added in 928c65a8cc5638b940bd2f6023aa02b3b0caac42